deploy/kubernetes/release/1.0/csi-datera-1.0.8.yaml

```diff
@@ -371,7 +371,7 @@ spec:
             capabilities:
               add: ["SYS_ADMIN"]
             allowPrivilegeEscalation: true
-          image: dateraiodev/dat-csi-plugin:v1.0.8
+          image: dateraiodev/dat-csi-plugin:v1.0.8-patch
```

Comment:
Does it need to be called patch or a minor version bump should be done here.

Reply:
New version 1.0.9 is tagged and released.